pom.xml

```diff
@@ -71,9 +71,9 @@
     <dependencies>
         <!--Bukkit API-->
         <dependency>
-                <groupId>org.bukkit</groupId>
-                <artifactId>bukkit</artifactId>
-                <version>1.15.2-R0.1-SNAPSHOT</version>
+                <groupId>org.spigotmc</groupId>
+                <artifactId>spigot-api</artifactId>
```

Comment:
This is changed due to the fact SpigotMC no longer publishes Bukkit jar files.

Source: https://www.spigotmc.org/threads/spigot-bungeecord-1-16-1.447405/

Reply:
Interesting.